babel/src/main/java/org/apache/calcite/sql/babel/SqlDateTimeAtLocal.java

```diff
@@ -29,13 +29,24 @@
 
 import java.util.List;
 
+/**
+ * A {@code SqlDateTimeAtLocal} is an AST node that describes
+ * the date time expression of At Local.
+ */
 public class SqlDateTimeAtLocal extends SqlCall implements SqlExecutableStatement {
   public static final SqlSpecialOperator OPERATOR =
       new SqlSpecialOperator("AT LOCAL", SqlKind.OTHER);
 
   private final SqlNode dateTimePrimary;
 
-  public SqlDateTimeAtLocal(SqlParserPos pos, SqlNode dateTimePrimary) {
+  /**
+   * Creates a {@code SqlTranslateUsingCharacterSet}.
```

Comment:
nit: Update @code to SqlDateTimeAtLocal

Reply:
Fix in https://github.com/googleinterns/calcite/pull/91/commits/043584683a05963e9c9bf5ec0b90f13175967577